tree/tree/test/TTree.cxx

```diff
@@ -0,0 +1,42 @@
+#include "TFile.h"
+#include "TTree.h"
+#include "TBranch.h"
+#include "TRandom.h"
+
+#include "gtest/gtest.h"
+
+
+class TTreeTest : public ::testing::Test {
+protected:
+    virtual void SetUp()
+    {
+        auto random = new TRandom(836);
+        auto file = new TFile("TTree.root", "RECREATE");
+        auto tree = new TTree("tree", "A test tree");
+        tree->SetAutoFlush(500);
```

Comment:
I don't think we actually need AutoFlush for this test (lets with very simple example)